asab/library/providers/filesystem.py

```diff
@@ -31,6 +32,13 @@ def __init__(self, library, path, layer, *, set_ready=True):
 		'''
 
 		super().__init__(library, layer)
+		# Parse file:// URI if present
+		if path.startswith('file:///'):
+			parsed_uri = urllib.parse.urlparse(path)
```

Comment:
I somehow like the idea of handling the path as url.
I even like the need to specify absolute paths only. 

But it seems to me more and more that both ideas do not fit here much.
The file path is not URL. It might not be URL-save and you get the path stripped. 

Like this path: `file:///my#library`
It is surely corner case... but

If you simply strip `file://`, you can support relative paths (e.g. `file://../mylibrary`). Which makes no sense in maestro and docker containers, cause the volume mapping would probably not fit when using relative path. However, I'd somehow expect it working in asab.